addons/link.py

```diff
@@ -18,7 +18,11 @@ class link(moduleBase):
                    'link-type' :
                         {'help' : 'type of link as in \'ip link\' command.',
                          'validvals' : ['dummy', 'veth'],
-                         'example' : ['link-type <dummy|veth>']}}}
+                         'example' : ['link-type <dummy|veth>']},
+                   'veth-peer-name':
+                        {'help' : 'Name of the veth peer interface.',
+                         'validvals' : '<interface>',
+                         'example' : ['veth-peer-name veth_ext2int']}}}
```

Comment:
I know its a long name. But all module attributes today follow a convention of prefixing the module name to the attribute. In this case can this be changed to link-veth-peer-name ?.